<a href="https://colab.research.google.com/github/Dasrunaik/AssignMents-ExcelR/blob/main/Assignment_4_Hypotheses.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import numpy as np
from scipy.stats import norm

In [ ]:
sample_mean = 3050
expected_x = 600
expected_mean = 1000 + 5 * expected_x    # Theoretical mean cost (μ = 4000)
std_dev_x = 25
std_dev_cost = 5 * std_dev_x            # Std dev of cost W
n = 25                              #sample size
alpha = 0.05

In [ ]:
standard_error = std_dev_cost / np.sqrt(n)             # SE = σ / √n
standard_error

np.float64(25.0)

In [ ]:
z_value = (sample_mean - expected_mean) / standard_error                # Z-test statistic
z_value

np.float64(-38.0)

Critical z-value for one-tailed test

In [ ]:
z_critical = norm.ppf(1 - alpha)
z_critical

np.float64(1.6448536269514722)

**Hypothesis Decision**

In [ ]:
if z_value > z_critical:
    decision = "Reject the null hypothesis"
    conclusion = "There is strong evidence that the weekly costs are higher than predicted."
else:
    decision = "Fail to reject the null hypothesis"
    conclusion = "There is not enough evidence to support the claim that costs are higher."

print(decision)
print(conclusion)


Fail to reject the null hypothesis
There is not enough evidence to support the claim that costs are higher.


**Conclusion:**

Theoretical mean = ₹4,000

Sample mean = ₹3,050

Z-score = -38.0 (approximately)

Critical value at α = 0.05 = 1.645



Result:Since the Z-score is much less than the critical value, we fail to reject the null hypothesis.